### PR TITLE
29 - Implement shortcut to insert external blocks in a toggle

### PR DESCRIPTION
**Pull request by juliansg022**
Developing #29.

### CODE REVIEW

**Comment by juliansg022** [comment]
Hi @juanhurtado10, @MarioRodriguezS, and @jcamilocp. This PR closes #29.

A listener was added to the editor to detect the `Tab` key in the editor to nest a block which is after a toggle. Now it works as the shortcut to nest a block inside a toggle in Notion.

**Comment by juliansg022** [comment]
This GIF can show how it works, preserving the features previously implemented in other issues:

Animation1

File: src/index.js [hunk 1]
```diff
@@ -533,4 +512,96 @@ export default class ToggleBlock {
...
+  createToggleWithShortcut() {
+    const redactor = document.activeElement;
+    redactor.addEventListener('keyup', (e) => {
+      if (e.code === 'Space') {
+        const blockContainer = document.activeElement;
+        const content = blockContainer.textContent;
+        const { length } = content;
+
+        if ((content[0] === '>') && (length - 1 === 1)) {
+          const invocatorBlock = this.api.blocks.getCurrentBlockIndex();
```
Comment (juanhurtado10) [inline src/index.js:529]: @juliansg022 what about using `blockCaller` instead?

File: src/index.js [hunk 2]
```diff
@@ -181,6 +161,8 @@ export default class ToggleBlock {
 
       // Event to add a block when the default content is clicked
       defaultContent.addEventListener('click', this.clickInDefaultContent.bind(this));
+
+      input.addEventListener('focus', this.setAttributesToNestBlock.bind(this));
```
Comment (juanhurtado10) [inline src/index.js:165]: @juliansg022 what about naming this function as`setNestedBlockAttributes`

**Review by juanhurtado10 — CHANGES_REQUESTED** [review]
@juliansg022 Good 👏  some minor comments were added!!

**Review by juanhurtado10 — APPROVED** [review]

**Pull request merged**